### PR TITLE
feat: able to calculate PLUR tvl

### PR DESCRIPTION
**Pull request by Chi-AnTai**
##### Name (to be shown on DefiLlama):
Plur

##### Twitter Link:
https://twitter.com/plurxyz

##### Website Link:
https://plur.xyz/

##### Logo (High resolution, will be shown with rounded borders):
https://whiteboard-shadow-box.s3.ap-northeast-1.amazonaws.com/eukhch1U.png

##### Current TVL:
105.32 k

##### Chain:
Blast

##### Short Description (to be shown on DefiLlama):
Plur is a platform for backing creators with no loss. You can create pools to earn native yield from the ETH staked by your followers and gain extra PLUR & Blast points.

##### Category (full list at https://defillama.com/categories) *Please choose only one:
SoFi

##### methodology (what is being counted as tvl, how is tvl being calculated):
We only accept WETH deposit at this moment, so TVL is the sum of all the WETH in our pools.

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/plur exports TVL: 
        
 
 
    blast                     105.02 k
    
    total                    105.02 k

File: projects/plur/index.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  return sumTokens2({
+    api: api,
+    owners: poolContracts,
+    token: WETH_ADDRESS
+  })
+}
+
+module.exports = {
+  timetravel: true,
+  start: BUILDER_CREATE_BLOCK,
```
Comment (g1nt0ki) [inline projects/plur/index.js:25]: this is unixTimestamp, not block number

File: projects/plur/index.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+const BUILDER_ADDRESS = '0x5dfee62C78A0E607CCE6A5d4458c328A03275ba2';
+const WETH_ADDRESS = "0x4300000000000000000000000000000000000004"
+const BUILDER_CREATE_BLOCK = 489054
+
+async function tvl(_, _1, _2, { api }) {
+  const createPoolLogs = await getLogs({
+    api,
+    fromBlock: BUILDER_CREATE_BLOCK,
+    target: BUILDER_ADDRESS,
+    topics: ["0x85e6dd046ca5129caaa517b25babd719637bb32968281fef3ede269234eb7250",],
```
[the remark this answers is not in the document]
Comment (g1nt0ki) [reply projects/plur/index.js:13]: also, TIL you can pass addreses as "0x000000000000000000000000a15fc23f780ae2c456b2d7fefedd8e77cda463f0", didnt know,m thanks for that

**Review by g1nt0ki — CHANGES_REQUESTED** [review]
.

**Comment by llamatester** [comment]
Error while running adapter at projects/plur: 
        
 
  
    
    
    ReferenceError: BUILDER_CREATE_BLOCK is not defined
    Truncated error stack:
    
        at tvl (/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/projects/plur/index.js:10:16)
        at getTvl (/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/test.js:50:29)
        at /home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/test.js:141:15
        at Array.map (<anonymous>)
        at /home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/test.js:129:29
        at Array.map (<anonymous>)
        at /home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/test.js:124:44

**Pull request merged**